packages/plugins/integration-zapier/src/lib/handlers/zapier-timer-started.handler.ts

```
@@ -0,0 +1,27 @@
+     * Handles the TimerStartedEvent by notifying Zapier webhooks
+     *
+     * @param event - The TimerStartedEvent that contains the time log details
+     * @returns A Promise that resolves once the webhooks are notified
+     */
+    async handle(event: TimerStartedEvent): Promise<void> {
+        const timeLog = event.timeLog;
+        await this.zapierWebhookService.notifyTimerStatusChanged({
+            event: 'timer.status.changed',
+            action: 'start',
+            data: timeLog,
+            tenantId: timeLog.tenantId!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)